app/src/modules/shell/shell.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/shell/shell.h>
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/zbus/zbus.h>
+
+#include "message_channel.h"
+
+LOG_MODULE_REGISTER(shell, CONFIG_APP_SHELL_LOG_LEVEL);
+
+static int cmd_zbus_ping(const struct shell *sh, size_t argc,
```

Comment:
Maybe the printing of `pong` could be implemented in a listener on a zbus channel so that the command `zbus ping` will actually send a message and if you get `pong` you know that zbus works

Reply:
nice, where is best? in trigger?

Reply:
@jtguggedal  Would that mean that we should add a separate internal channel (ZBUS_TEST_CHANEL perhaps) here and publish the ping to the channel and then respond back from a receiver in the same file?

Reply:
should I just add it here or in separate PR?

Reply:
I'd just to it here, but follow-up PR is fine too